packages/components/src/Dialog/DialogTrigger.tsx

```diff
@@ -1,40 +1,46 @@
-import { ReactNode, createContext } from 'react';
+import type { ReactNode } from 'react';
 import { DialogTrigger } from 'react-aria-components';
 import type RAC from 'react-aria-components';
 import { PressResponder } from '@react-aria/interactions';
+import type { DialogContextProps } from './Context';
+import { DialogContext } from './Context';
 
+// Props
+// ---------------
 type RemovedProps =
   | 'children'
   | 'isOpen'
   | 'isDismissable'
   | 'isKeyboardDismissDisabled';
-export interface ModalProps extends RAC.ModalOverlayProps {
-  open?: boolean;
-  dismissable?: boolean;
-  keyboardDismissable?: boolean;
-  size?: string;
-}
 
 export interface DialogTriggerProps
   extends Omit<RAC.DialogTriggerProps, 'isOpen'>,
-    Omit<ModalProps, RemovedProps> {}
+    Omit<DialogContextProps, RemovedProps> {
```

Comment:
some props like `dismissable`, or `keyboarddismissable` have no description in the props table